frontend/src/api/k8s/inferenceServices.ts

```diff
@@ -23,37 +24,40 @@ const assembleInferenceService = (
   const { path, dataConnection } = storage;
   const dataConnectionKey = secretKey || dataConnection;
 
-  return {
-    apiVersion: 'serving.kserve.io/v1beta1',
-    kind: 'InferenceService',
-    metadata: {
-      name,
-      namespace: project,
-      labels: {
+  return addOwnerReference(
```

Comment:
Just to sync this with you, I assume you are expecting that we add the same in `assembleServingRuntime` for kserve right? So we have a circular reference just for that platform right?

Reply:
I've removed the owner reference on inference service for now. So no we will not need a change to `assumbleServingRuntime` right now.